library/blob.c

```diff
@@ -3372,10 +3372,14 @@ struct eblob_backend *eblob_init(struct eblob_config *c)
 	INIT_LIST_HEAD(&b->bases);
 	b->max_index = -1;
 
+	err = pthread_rwlock_init(&b->iteration_lock, NULL);
```

Comment:
Please, move this init after `inspect_lock` initialization

Reply:
done